src/openloco/windows/stationwnd.cpp

```diff
@@ -24,7 +24,7 @@ using namespace openloco::map;
 
 namespace openloco::ui::windows::station
 {
-    static loco_global<uint16_t[0x24000], 0x000F00484> _byte_F00484;
+    static loco_global<uint8_t[0x24000], 0x00F00484> _byte_F00484;
```

Comment:
Hmm, a `uint8_t` seems a bit small for the map coordinate system. I assume this is related to showing the catchment area tiles? Perhaps the issue is with the length rather than the type?

Reply:
From what I can see in the assembly only bit 0 and 1 are ever set in _byte_F00484. Also there is no multiplier when indexing into the array.

Reply:
That makes sense then. Length-wise, it makes sense as well: 384*384 = 0x24000

So it's probably two flags for every tile on the map.

Reply:
We have a constant for this already. I suggest we use it:

```suggestion
    static loco_global<uint8_t[map::map_size], 0x00F00484> _byte_F00484;
```

NB: it's defined as `map_size = map_columns * tile_size`, without mention of 0x24000. I suppose this made it harder to find.

Reply:
I've changed map_size to `int32_t map_size = map_columns * map_rows` as the previous value of map_size was 0x3000 not 0x24000. I created map_height and map_width to work with tile_loop instead.